configure

```diff
@@ -929,6 +952,8 @@ export PATH     := \$(EXTLIB)/bin:\$(PATH)
 export CFLAGS   := -I\$(EXTLIB)/include \$(CFLAGS) $EXTRACFLAGS
 export CPPFLAGS := -I\$(EXTLIB)/include \$(CPPFLAGS) $EXTRACFLAGS
 export LDFLAGS  := -L\$(EXTLIB)/lib \$(LDFLAGS) $EXTRALDFLAGS
+export DISCORDFLAGS := $DISCORDFLAGS
```

Comment:
Do you really need separate flags? Just add them to extra in my opinion.

Reply:
Makes sense